src/server.h

```diff
@@ -123,7 +124,8 @@ struct ServerSoundParams
 		pos(0,0,0),
 		object(0),
 		max_hear_distance(32*BS),
-		loop(false)
+		loop(false),
+		fade(0)
```

Comment:
So is `gain` , `pos` and `max_hear_distance` initialized the wrong way too?
Could we leave this untouched and only change it for calculations/comparisons?